discovery-kubernetes-api/src/main/scala/akka/discovery/kubernetes/JsonFormat.scala

```diff
@@ -18,7 +18,8 @@ import spray.json._
   implicit val containerFormat: JsonFormat[Container] = jsonFormat2(Container.apply)
   implicit val podSpecFormat: JsonFormat[PodSpec] = jsonFormat1(PodSpec.apply)
   implicit val containerStatusFormat: JsonFormat[ContainerStatus] = jsonFormat2(ContainerStatus.apply)
-  implicit val podStatusFormat: JsonFormat[PodStatus] = jsonFormat3(PodStatus.apply)
+  implicit val podConditionFormat: JsonFormat[PodCondition] = jsonFormat2(PodCondition.apply)
```

Comment:
Part of me kind of would like to turn the conditions into a `Map` and do some magic in the spray format, but

* I'm not totally up on my spray-fu
* the convention here is to mirror the JSON structure directly